### ISSUE TITLE
Geojson-vt

### ISSUE THREAD

**Issue by mb72** [tomchadwin/qgis2web#728]
Could Geojson-vt be incorporated to allow much larger geojson files?

github.com/mapbox/geojson-vt

**Comment by tomchadwin** [comment]
Yes, potentially. I've not looked into it through unfamiliarity with it - eg, does it still allow labelling/popups/highlight on hover? I guess what I need to know is whether it can be a simply option for vector layers, or whether it has other effects.

Contributions extremely welcome,..

**Comment by tomchadwin** [comment]
So as far as I can see, the following steps would be required to get this to work:

1. Include Leaflet.VectorGrid (no problem - already used for external VTs)
2. Use LVG.slicer to create the layer
3. Style using the LVG syntax, not normal Leaflet (I can't get this to work manually yet on a point dataset)
4. Add a checkbox in the dialog to specify on-the-fly VT

I'm unsure what knock-on effects this might have on eg layer interactivity, labels, and popups, but even if we can't have those, it's another option for users.
